### ISSUE TITLE
No text copy

### ISSUE THREAD

**Issue by mipastgt** [dlemmermann/jfxcentral#37]
Jfx-Central does not allow copying any text. This makes it, e.g., impossible to get at the original link of the JavaFX flags. It is not marked as a link that you can click on and you cannot even copy/paste the link into a browswer. I assume nobody is going to type it in. All text on the web-site should be copyable.

**Comment by dlemmermann** [comment]
@FlorianKirmaier can we?

**Comment by FlorianKirmaier** [comment]
Yes, it's configurable in JPro. I will take a look, when I announce the new scrolling.
